lib/bitreserve/request_data.rb

```diff
@@ -0,0 +1,3 @@
+module Bitreserve
+  RequestData = Struct.new(:endpoint, :entity, :client)
+end
```

Comment:
I'm not sure how I feel about this.
If the goal is to reduce the number of arguments for each method, wouldn't one of the following be a better approach?

1. Make each entity know about it's own endpoint, rather than centralize them all in an Endpoints module
2. Find a way to now have to send `self` (the client) for each request, which seems a bit weird to be honest